src/widgets/favoritewidget.cpp

```diff
@@ -50,12 +50,16 @@ QSize FavoriteWidget::sizeHint() const {
 }
 
 void FavoriteWidget::paintEvent(QPaintEvent* e) {
-  QStylePainter p(this);
+  // ignore text descent when vertically centering
+  QFontMetrics fontMetrics = QFontMetrics(this->font());
+  QRect rect = this->rect();
+  rect.setBottom(rect.bottom() - fontMetrics.descent());
```

Comment:
I'm afraid the height of `this->rect()`is not yet defined (or at least does not have the final value) when calling the constructor.
I'm not too familiar with Qt and/or Clementine code myself but I'm open for suggestions if you have another idea...